buckw

```diff
@@ -66,7 +66,7 @@ OKBUCK_DIR="$SCRIPT_DIR/.okbuck"
 MAX_DISPLAY_CHANGES=10
 WATCHMAN_TIMEOUT=10
 WATCHMAN_FAILED="WATCHMAN FAILED"
-BUCK_BINARY_DIR="$OKBUCK_DIR/cache/buck_binary"
+BUCK_BINARY_DIR="3rdparty/com/github/facebook"
```

Comment:
Can we let users customize this `3rdparty` bit? It might conflict with an existing dir they have.

Reply:
sure

Reply:
this will go away. working on moving buck download as a special tool task